exit.go

```diff
@@ -39,27 +39,40 @@ type ExitConfiguration struct {
 	SubscriptionFee      string                     `json:"SubscriptionFee"`
 	ClaimInterval        uint32                     `json:"ClaimInterval"`
 	Services             map[string]ExitServiceInfo `json:"Services"`
+	ReverseNanoPayFee    string                     `json:"ReverseNanopayfee"`
 }
 
 type TunaExit struct {
-	config           *ExitConfiguration
-	wallet           *nkn.Wallet
-	services         []Service
-	serviceConn      *cache.Cache
-	common           *Common
-	tcpListener      net.Listener
-	udpConn          *net.UDPConn
-	reverseIp        net.IP
-	reverseTcp       []int
-	reverseUdp       []int
-	onEntryConnected func()
-	closeChan        chan struct{}
+	config              *ExitConfiguration
+	Wallet              *nkn.Wallet
```

Comment:
It's better to remove wallet and use wallet in common to be consistent with entry